src/api/index.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * This is a mocked API so that we don't have to have any backend data to start with.
+ * But we do write this as a set of API Calls so that it won't be complicated to
+ * re-factor it if we ever need to have a backend.
+ */
+
+import ApiFactory from './api';
+
+import StopData, { IStop } from './stops';
+import RouteData, { IRoute } from './routes/index';
+
```

Comment:
Im still a little confused on this factory. It seems here the constants for `Stops` and `Routes` are expecting actual data, not a reference to the api... (I hope thats not a dumb comment, I am still rusty with factories in the wild to be honest). Should `ApiFactory` be renamed to `DataFactory`? 
(And maybe typing the constants will help make it clearer too?)

Reply:
OK, OK, I get it now, lol. `Stops` and `Routes` are inside `ApiFactory` so they are implied to be APIs. Thanks for bearing with my slow brain on that one!

Reply:
Yep, although currently just dumb json data. If/when we eventually use a backend api, this will be much easier to re-factor instead of having to re-write some of the functionalities in the UI as well.